newhelm/sut_decorator.py

```diff
@@ -1,16 +1,20 @@
 from functools import wraps
 import inspect
+from typing import Sequence, Type
 from newhelm.record_init import add_initialization_record
 from newhelm.sut import SUT
+from newhelm.sut_capabilities import SUTCapability
 
 
-def newhelm_sut():
+# TODO: Remove defaulting.
+def newhelm_sut(capabilities: Sequence[Type[SUTCapability]] = []):
```

Comment:
I had the decorator take in the `capabilities` to require that every subclass sets them uniquely. An alternative would be to have the class set `capabilities` directly, then have the decorator verify that `capabilities` is set.

Reply:
Another option that omits the need for the decorator is to require SUTs to report their capabilities through an abstract method, similar to the way test dependencies work. But I also don't mind the decorator approach.